src/cpp/proxy/sockjssession.cpp

```diff
@@ -244,7 +244,6 @@ class SockJsSession::Private : public QObject
 		}
 		requests.clear();
 
-		wsConnectionMap.erase(sock);
```

Comment:
should probably still disconnect here somehow?

the connections object seems to have a default constructor, so maybe just:
```
wsConnection = WSConnections();
```